### PR TITLE
Josh/create josh activity

### PR DESCRIPTION
**Pull request by JoshGoldsmith**
Approve this. it's good i swear

### CODE REVIEW

File: app/src/main/java/jayp/bge/playboikotlin/JoshActivity.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+
+class JoshActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_josh)
+
+        val button: Button = findViewById(R.id.joshButton)
```
Comment (zohebaziz) [inline app/src/main/java/jayp/bge/playboikotlin/JoshActivity.kt:12]: goodButton

File: app/src/main/java/jayp/bge/playboikotlin/JoshActivity.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+        button.setOnClickListener {
+            finish()
```
Comment (zohebaziz) [inline app/src/main/java/jayp/bge/playboikotlin/JoshActivity.kt:15]: on what

File: app/src/main/java/jayp/bge/playboikotlin/MainActivity.kt [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+
+            val intent = Intent(this, ZebActivity::class.java)
+
+            startActivity(intent)
+        }
+
+        val button: Button = findViewById(R.id.joshButton)
+
+        button.setOnClickListener {
+            val i = Intent(this@MainActivity, JoshActivity::class.java)
```
Comment (zohebaziz) [inline app/src/main/java/jayp/bge/playboikotlin/MainActivity.kt:33]: advanced kotlin !

File: app/src/main/res/layout/activity_josh.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/joshButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Josh Button"
```
Comment (zohebaziz) [inline app/src/main/res/layout/activity_josh.xml:23]: josh button josh button

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -6,13 +6,45 @@
...
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <Button
+        android:id="@+id/zebButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Zeb Button"
```
Comment (zohebaziz) [inline app/src/main/res/layout/activity_main.xml:35]: yo dis me tho ?

File: app/src/main/java/jayp/bge/playboikotlin/MainActivity.kt [hunk 6]
```diff
@@ -1,11 +1,39 @@
...
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val robbieButton: Button = findViewById(R.id.robbieButton)
+        robbieButton.setOnClickListener {
+
+            val intent = Intent(this, RobTivity::class.java)
+            startActivity(intent)
+
+        }
```
Comment (robert-chatterton) [inline app/src/main/java/jayp/bge/playboikotlin/MainActivity.kt:19]: welcome to what a good button looks like

**Pull request merged**